diagnostic.md

```diff
@@ -44,9 +56,12 @@ How would we reference each of the following categories of elements?
 -   Only those elements belonging to both the `important` and `red` _classes_
 -   All `p` elements whose parent elements are `div`s
 
-<!-- your answer starts here -->
-
-<!-- your answer ends here -->
+```CSS
+.big
+#contentPane
+.important, .red
+div p
```

Comment:
Close! This will select any p that is a descendant of a div, at any level. We want only p's that have a div as a direct parent: `div > p`